undertow/src/main/java/org/wildfly/extension/undertow/AccessLogDefinition.java

```diff
@@ -56,7 +56,7 @@
             .setDefaultValue(new ModelNode("default"))
             .build();
     protected static final SimpleAttributeDefinition PREFIX = new SimpleAttributeDefinitionBuilder(Constants.PREFIX, ModelType.STRING, true)
-            .setDefaultValue(new ModelNode("access_log"))
+            .setDefaultValue(new ModelNode("access_log."))
```

Comment:
Yes, it looks like having the '.' in both PREFIX and SUFFIX may cause a problem. In io.undertow.server.handlers.accesslog.DefaultAccessLogReceiver the value of defaultLogFile will be "access_log..log".